### DISCUSSION TITLE
PCL在没给控件的情况下多出了控件报错

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#4424] [提问] [answered]
![42c70f729f41c5dfa69734ee0f24ad9d](https://github.com/user-attachments/assets/6c0cdcd0-6217-4c12-826d-a5b1fab0fde7)
![cabf633fc8771207081da0f52c26c496](https://github.com/user-attachments/assets/721485de-117e-460a-ae20-73ee564bdebf)
这是为啥
我也没写上述的StackPanel

**Comment by Joker2184** [comment]
MyHint就是第一行 前面没东西了

**Comment by MoYuan-CN** [comment]
发一下你的完整 XAML 文件，我分析一下

**Comment by Joker2184** [comment, marked as answer]
重构后正常
